scalajslib/worker/1/src/mill/scalajslib/worker/ScalaJSWorkerImpl.scala

```diff
@@ -150,14 +145,15 @@ class ScalaJSWorkerImpl extends ScalaJSWorkerApi {
             )
         else withModuleSplitStyle
 
-      StandardImpl.clearableLinker(withOutputPatterns)
+      val linker = StandardImpl.clearableLinker(withOutputPatterns)
+      val irFileCache = StandardImpl.irFileCache().newCache
```

Comment:
The IRFileCache is capable to cache IR across different linker instances.

So ideally, you have a single instance of `irFileCache()` in the entire build (and a `newCache` per linker). This ensures that, for example, the Scala library is loaded only once. Otherwise you might see very high memory usage.